test/common.js

```diff
@@ -598,8 +598,7 @@ exports.WPT = {
     try {
       fn();
     } catch (err) {
-      if (err instanceof Error)
-        err.message = `In ${desc}:\n  ${err.message}`;
+      console.error(`In ${desc}:`);
```

Comment:
Hmm, yes, this probably deserves another PR to sort out. I am fine with a `console.error` work around.